compiler-core/src/warning.rs

```diff
@@ -355,7 +366,29 @@ To match on all possible lists, use the `_` catch-all pattern instead.",
                     extra_labels: vec![],
                 }),
             },
-
+            Warning::DeprecatedSyntax {
+                path,
+                src,
+                warning: DeprecatedSyntaxWarning::DeprecatedEmptyGuardClause { location },
+            } => Diagnostic {
+                title: "Deprecated empty guard clause syntax".into(),
+                text: wrap(
+                    "Empty clause within a guard is deprecated.
```

Comment:
This is just a guard, not a clause within a guard. Also make sure to escape newlines so the `wrap` function works properly.

```suggestion
                    "This syntax for an empty guard is deprecated. \
```